From 5293d0f5067b2aeefe9ce3c175c972de367589bc Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Wed, 13 Feb 2019 17:43:54 +0100
Subject: [PATCH] modules/windows: split WINDRES env variable

As it may contain not just the binary, but also the arguments to it.

Upstream-Status: Pending
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 mesonbuild/modules/windows.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mesonbuild/modules/windows.py b/mesonbuild/modules/windows.py
index d185d89..dbaeb9b 100644
--- a/mesonbuild/modules/windows.py
+++ b/mesonbuild/modules/windows.py
@@ -56,7 +56,7 @@ class WindowsModule(ExtensionModule):
             if 'WINDRES' in os.environ:
                 # Pick-up env var WINDRES if set. This is often used for
                 # specifying an arch-specific windres.
-                rescomp = ExternalProgram('windres', command=os.environ.get('WINDRES'), silent=True)
+                rescomp = ExternalProgram('windres', command=os.environ.get('WINDRES').split(), silent=True)
 
         if not rescomp or not rescomp.found():
             # Take windres from the config file after the environment, which is
